external_parser/metrics/metrics.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace metrics {
+struct joiner_metrics {
+  int number_of_learned_events = 0;
```

Comment:
are you planning on adding the skip events on this PR?

Reply:
added